lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -780,9 +786,15 @@ def create_build_phase_to_symlink_header_folders(native_target)
           #         The consumer for the specification for which the compiler flags
           #         are needed.
           #
+          # @param  [Boolean] arc
+          #         Whether the arc is enabled or not.
+          #
+          # @param  [Symbol] language
+          #         The language these compiler warnings are for. Can be either :objc or :swift.
+          #
           # @return [String] The compiler flags.
           #
-          def compiler_flags_for_consumer(consumer, arc)
+          def compiler_flags_for_consumer(consumer, arc, language)
             flags = consumer.compiler_flags.dup
             if !arc
               flags << '-fno-objc-arc'
```

Comment:
I wasnt sure if this makes sense for Swift files but Xcode 11 Beta 4 allows it.

Reply:
huh, didn't even think of that. that's interesting

Reply:
actually that seems to be a lie. investigating more.

Reply:
nope, not allowed either.

Reply:
updated with a test to remove `-fno-objc-arc` for Swift source files.